features/signup.feature

```diff
@@ -0,0 +1,10 @@
+Feature: signup
+
+          as a user 
+          I want to create a user
+          in order to validate figure el pop up Sign up successful
+
+ Scenario: sign up validate pop up 
+        Given the user is on https://www.demoblaze.com/
+        When user must register with non-existing user
```

Comment:
este step esta mal definido
en ingles estas diciendo esto:
el usuario debe registrarse con un usuario no existente (parece como si fuera la declaracion de una precondicion) no refleja la necesidad del negocio de controlar esta feature, que es lo que se detalla en un gherkin. 

deberia decir algo como esto:
el usuario se registra con credenciales no registradas en el sistema.
the user signs up with credentials not registered on the system. 
(que en realidad tampoco te hace falta tener este nivel de detalle, pero si lo queres agregar seria de esa forma)

lo que me interesa que manejes 100% es la forma en la que se escriben los tests cases manuales, porque a eso te vas a tener que dedicar cuando entres a laburar a alguna empresa.